CHANGELOG.md

```diff
@@ -48,6 +48,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 Here is an overview of all **stable** additions:
 
+- **General**: Add TLS support to RabbitMQ scler ([#967](https://github.com/kedacore/keda/issues/967))
```

Comment:
Move this to improvements section because this is more an improvement in rabbitmq scaler than a new thing.
WRT the style guide, please use something like:
`- **RabbitMQ Scaler**: Add TLS support ([#967](https://github.com/kedacore/keda/issues/967))`

Reply:
Ok, moved it to improvements section.